src/main/java/frc/robot/Robot.java

```diff
@@ -162,7 +162,7 @@ private void configureBindings() {
         .driverController
         .leftTrigger()
         .onTrue(robotCommands.intakeCommand())
-        .onFalse(robotCommands.stowCommand());
+        .onFalse(robotCommands.intakeAssistCommand());
```

Comment:
these are mixed around, we want holding the button to be assist, and clicking to be regular intake